packages/web/src/lib/web-mode.ts

```diff
@@ -261,7 +261,10 @@ export class WebMode {
 					return { txn: txn, msig: msig.mparams }; // multi singature
 				}
 				default:
-					return { txn: txn, authAddr: execParams[txnId].fromAccount?.addr }; // set signer
+					return {
+						txn: txn,
+						authAddr: execParams[txnId].fromAccount?.addr || execParams[txnId].fromAccountAddr
+					}; // set signer
```

Comment:
Is it possible to add a test?

Reply:
we already have a test for algosigner webmode, the common conventions is to use `fromAccount: {addr:"",sk:""}`, but sometimes user also use `fromAccountAddr`, thus added it

Reply:
Nah, I don't think so because the wallet actually wait for the cosigner signature, and I don't think it is possible with unit test, it can done from the wallet UX only